cristal-integration-webjar/src/main/node/langs/translation-en.json

```diff
@@ -0,0 +1,3 @@
+{
+  "xwiki.switch.action.title": "Switch to XWiki"
```

Comment:
We'll need to registrer those translations in weblate even if Cristal doesn't yet allow switching locale.

Reply:
Created https://l10n.xwiki.org/projects/xwiki-contrib/cristal-integration-webjar/